Controller/FolderController.php

```diff
@@ -29,19 +29,8 @@ class FolderController extends Controller
      * @throws NotFoundHttpException $location is flagged as invisible
```

Comment:
I think @clash82 suggested this can be removed as well now, same with the other method and then also the `use` line at the top.